legend-engine-core/legend-engine-core-executionPlan-execution/legend-engine-executionPlan-execution/src/test/java/org/finos/legend/engine/plan/execution/parameterization/TestParameterizedValueSpecification.java

```diff
@@ -69,6 +69,30 @@ public void testLambdaIsParameterized() throws IOException
 
     }
 
+    @Test
+    public void testStringListIsNotParameterized() throws IOException
```

Comment:
can we/should write a test using Pure code/grammar ... code->PMCD->compile->PureModel.getFunction ? ... this is pretty hard to follow